packages/babel-core/src/transformation/block-hoist-plugin.js

```diff
@@ -32,7 +32,8 @@ const blockHoistPlugin = {
    *  - 1 Default node position
    *  - 2 Priority over normal nodes
    *  - 3 We want this to be at the **very** top
-   *  - 4 Reserved for the helpers used to implement module imports.
+   *  - 4 Imports, which must be available before the code is executed.
+   *  - 5 Reserved for the helpers used to implement module imports.
```

Comment:
could we possibly name those blocks and use them as constants through the code? those are really magic numbers and I had no idea for a while what they stand for and where to find info about it quickly when I've started looking though the babel's codebase

Reply:
lol js hoisting is like `z-index`

Reply:
🤣

Reply:
Do yuo have a good suggestion for a better name for `A_LITTLE_BIT_MORE`?

```js
export const BOTTOM = 0;
export const DEFAULT = 1;
export const A_LITTLE_BIT_MORE = 2;
export const TOP = 3;
export const IMPORT = 4;
export const MODULE_HELPER = 5;
```

Reply:
Also I really don't know where to export that constants from.
`import ... from "@babel/core"`? `from "@babel/traverse"`? `from "@babel/core/lib/transformation/block-hoist-plugin"`? Or maybe `from "@babel/helper-block-hoist"`?

I'm continuing to switch between one option and the others lol.